### ISSUE TITLE
Mod could use some lag fixes

### ISSUE THREAD

**Issue by LemADEC** [mallrat208/Re-WIRED#9]
As of 1.0.4, the mod is causing a fair amount of lag:
![image](https://user-images.githubusercontent.com/4776032/61159402-33d7b800-a4fc-11e9-8d37-85cf4e2ea039.png)

The Cyberware mod itself had a lot of refactoring to fix its own lag.
If we look with a profiler, we can see about 50% from the Cyberware event lag actually comes from Re-WIRED now:
![image](https://user-images.githubusercontent.com/4776032/61159614-e3ad2580-a4fc-11e9-935b-36cd73962739.png)

Here, the most visible lag is from the constant creation of ItemStacks.
After that, there's also lag coming from repetitive calls to CyberwareAPI.getCyberware() which is fairly extensive and needs to be cached. The CyberwareAPI is being updated to help you with fixing that.
Long story short, Re-WIRED could use an update :). How would you prefer to proceed?
